src/doc/CreateHighlightDialog.js

```diff
@@ -69,10 +69,7 @@ class CreateHighlightDialog extends EventEmitter {
 
     unmountPopover() {
         this.isVisible = false;
-        const pageEl = shouldDisplayMobileUI(this.container)
-            ? this.container
-            : getPageEl(this.annotatedElement, this.location.page);
-        const popoverLayer = pageEl.querySelector('.ba-dialog-layer');
+        const popoverLayer = this.container.querySelector('.ba-dialog-layer');
```

Comment:
could you use `getPopoverParent` to get pageEl here?

Reply:
Not in this case. We just need to get the dialog layer from the container. We don't need a specific page number to get that :)